Docs/feature_memory_protection.md

```diff
@@ -401,3 +401,66 @@ manner similar to below:
 ```
 
 before building the HOB.
+
+## **Memory Protection Special Regions**
+
+Memory protection is not activated until the CPU Architecture Protocol has been installed because the
+protocol facilitates access to the attribute set functions in CpuDxe. Many allocations and image loads

```

Comment:
"facilitates access to the attribute set functions in CpuDxe."

I think this should mention that the "attribute set function in CpuDxe" means that it goes to the Page Tables.

Reply:
Updated